packages/fields/src/Implementation.js

```diff
@@ -4,23 +4,24 @@ import { parseFieldAccess } from '@keystone-alpha/access-control';
 class Field {
   constructor(
     path,
-    config,
+    { hooks = {}, isRequired, defaultValue, access, label, schemaDoc, ...config },
```

Comment:
This is the crux of this PR. In a future PR I intend to look at how the remaining config options are used in relation to the field adapters.